task-json/countries.js

```diff
@@ -0,0 +1,270 @@
+    {name: 'Romania', code: 'RO'}, 
+    {name: 'Russian Federation', code: 'RU'}, 
+    {name: 'RWANDA', code: 'RW'}, 
+    {name: 'Saint Helena', code: 'SH'}, 
+    {name: 'Saint Kitts and Nevis', code: 'KN'}, 
+    {name: 'Saint Lucia', code: 'LC'}, 
+    {name: 'Saint Pierre and Miquelon', code: 'PM'}, 
+    {name: 'Saint Vincent and the Grenadines', code: 'VC'}, 
+    {name: 'Samoa', code: 'WS'}, 
+    {name: 'San Marino', code: 'SM'}, 
+    {name: 'Sao Tome and Principe', code: 'ST'}, 
+    {name: 'Saudi Arabia', code: 'SA'}, 
+    {name: 'Senegal', code: 'SN'}, 
+    {name: 'Serbia and Montenegro', code: 'CS'}, 
+    {name: 'Seychelles', code: 'SC'}, 
+    {name: 'Sierra Leone', code: 'SL'}, 
+    {name: 'Singapore', code: 'SG'}, 
+    {name: 'Slovakia', code: 'SK'}, 
+    {name: 'Slovenia', code: 'SI'}, 
+    {name: 'Solomon Islands', code: 'SB'}, 
+    {name: 'Somalia', code: 'SO'}, 
+    {name: 'South Africa', code: 'ZA'}, 
+    {name: 'South Georgia and the South Sandwich Islands', code: 'GS'}, 
+    {name: 'Spain', code: 'ES'}, 
+    {name: 'Sri Lanka', code: 'LK'}, 
+    {name: 'Sudan', code: 'SD'}, 
+    {name: 'Suriname', code: 'SR'}, 
+    {name: 'Svalbard and Jan Mayen', code: 'SJ'}, 
+    {name: 'Swaziland', code: 'SZ'}, 
+    {name: 'Sweden', code: 'SE'}, 
+    {name: 'Switzerland', code: 'CH'}, 
+    {name: 'Syrian Arab Republic', code: 'SY'}, 
+    {name: 'Taiwan, Province of China', code: 'TW'}, 
+    {name: 'Tajikistan', code: 'TJ'}, 
+    {name: 'Tanzania, United Republic of', code: 'TZ'}, 
+    {name: 'Thailand', code: 'TH'}, 
+    {name: 'Timor-Leste', code: 'TL'}, 
+    {name: 'Togo', code: 'TG'}, 
+    {name: 'Tokelau', code: 'TK'}, 
+    {name: 'Tonga', code: 'TO'}, 
+    {name: 'Trinidad and Tobago', code: 'TT'}, 
+    {name: 'Tunisia', code: 'TN'}, 
+    {name: 'Turkey', code: 'TR'}, 
+    {name: 'Turkmenistan', code: 'TM'}, 
+    {name: 'Turks and Caicos Islands', code: 'TC'}, 
+    {name: 'Tuvalu', code: 'TV'}, 
+    {name: 'Uganda', code: 'UG'}, 
+    {name: 'Ukraine', code: 'UA'}, 
+    {name: 'United Arab Emirates', code: 'AE'}, 
+    {name: 'United Kingdom', code: 'GB'}, 
+    {name: 'United States', code: 'US'}, 
+    {name: 'United States Minor Outlying Islands', code: 'UM'}, 
+    {name: 'Uruguay', code: 'UY'}, 
+    {name: 'Uzbekistan', code: 'UZ'}, 
+    {name: 'Vanuatu', code: 'VU'}, 
+    {name: 'Venezuela', code: 'VE'}, 
+    {name: 'Viet Nam', code: 'VN'}, 
+    {name: 'Virgin Islands, British', code: 'VG'}, 
+    {name: 'Virgin Islands, U.S.', code: 'VI'}, 
+    {name: 'Wallis and Futuna', code: 'WF'}, 
+    {name: 'Western Sahara', code: 'EH'}, 
+    {name: 'Yemen', code: 'YE'}, 
+    {name: 'Zambia', code: 'ZM'}, 
+    {name: 'Zimbabwe', code: 'ZW'} 
+  ];
+  function list_countries(){
+    const listcountries=[];
+    for (var key in countries) {
+         let obj =JSON.parse(key);
+        listcountries.push(countries[obj].name);
+  }
+  return listcountries;

```

Comment:
try to use prebuilt methods
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

Reply:
i dont get it here to be honest what should i use foreach ? or arrays.keys its still the same

Reply:
I suggest to read the whole documentation in the link, but the function that you should use is `.map()`